CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/PopupMenu` title to marquee for long text
 - `agate/RadioItem` icons to not be bigger than icon container
 - `agate/TabGroup` button padding for Cobalt and Copper skins
+- `agate/TemperatureControl` to have text inline for Spanish locale
```

Comment:
move changelog line to unreleased section

Reply:
Done